config/default.json

```diff
@@ -6,5 +6,7 @@
     "default": 10,
     "max": 50
   },
-  "mongodb": "mongodb://localhost:27017/gamification"
+  "rabbitQueue": "my-queue",
+  "rabbitExchange": "my-exchange",
+  "rabbitRoutingKey": "testkey"
```

Comment:
Wouldn't it make sense to name this `my-key` in correspondence with the other names in this file?